app/src/main/res/values/strings.xml

```diff
@@ -650,24 +650,24 @@ Before uploading your changes, the app checks with a &lt;a href=\"https://www.we
 
     <string name="quest_bicycleParkingCoveredStatus_title">"Is this bicycle parking covered (protected from rain)?"</string>
 
-    <string name="quest_bicycle_parking_fee_title">Do you have to pay to park a bike here?</string>
+    <string name="quest_bicycle_parking_fee_title">Do you have to pay to park a bicycle here?</string>
 
     <string name="quest_bicycle_parking_type_title">What type of bicycle parking is this?</string>
-    <string name="quest_bicycle_parking_type_stand">Stand (supports bike frame)</string>
+    <string name="quest_bicycle_parking_type_stand">Stand (supports bicycle frame)</string>
     <string name="quest_bicycle_parking_type_wheelbender">Wheelbender (supports wheel only)</string>
     <string name="quest_bicycle_parking_type_shed">Shed</string>
     <string name="quest_bicycle_parking_type_locker">Locker</string>
     <string name="quest_bicycle_parking_type_building">Building</string>
     <string name="quest_bicycle_parking_type_handlebarholder">Handlebar holder</string>
 
 
-    <string name="quest_bicycle_rental_capacity_title">How many rental bike spaces are here?</string>
+    <string name="quest_bicycle_rental_capacity_title">How many rental bicycle spaces are here?</string>
 
     <string name="quest_bicycle_rental_type_title">What type of bicycle rental is this?</string>
-    <string name="quest_bicycle_rental_type_docking_station">Docking station which locks bikes</string>
-    <string name="quest_bicycle_rental_type_dropoff_point">Designated spot (bikes locked individually)</string>
-    <string name="quest_bicycle_rental_type_human">Human-operated bike rental</string>
-    <string name="quest_bicycle_rental_type_shop_with_rental">Bike store that also rents bikes</string>
+    <string name="quest_bicycle_rental_type_docking_station">Docking station which locks bicycles</string>
+    <string name="quest_bicycle_rental_type_dropoff_point">Designated spot (bicycles locked individually)</string>
+    <string name="quest_bicycle_rental_type_human">Staffed bicycle rental</string>
```

Comment:
* Staffed, not human-operated.  Nobody would ever say "human-operated" 😂 
* Bike would be the most common use, but because of the confusion with motorcycle ("bike" is shorthand for "motorcycle" in en_US), it's always acceptable to explicitly say bicycle.

Reply:
🤖